src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java

```diff
@@ -220,6 +222,22 @@ public int getWascId() {
 	 */
 	private boolean findSourceCodeSVN(HttpMessage originalMessage) throws Exception {
 
+		//if we got a 404 or a redirect specifically, then this is NOT a match
+		//note that since we are simply relying on the file existing or not, we 
+		//will not attempt any fuzzy matching. Old school.
+		//Checks based on this are necessary, otherwise a recursive scan on nodes 
+		//in the url path cause lots of false positives.
+		List<HttpStatusCode> unwantedCodes= new ArrayList<HttpStatusCode>();
```

Comment:
When checking the "contains", it will use the status code which is an int.
The class HttpStatusCode is just an utility class (used to gather all the status codes and provide helper methods).